recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.24.0" %}
+{% set version = "0.24.1.dev0" %}
 
 package:
   name: featuretools
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/f/featuretools/featuretools-{{ version }}.tar.gz
-  sha256: 32a7f1ae84ababa757a46ccef1695f0843b0f224c5aae4a1ddfffb6e4c033351
+  sha256: fa0b444055f8bb4046d79823f4b0226b740cbc1ad1016b2ca80c35138168ec29
 
 build:
   entry_points:
```

Comment:
We made changes to requirements.txt, and test-requirements.txt in Featuretools, and those should be reflect here.
For example, scipy min was upgraded, pandas min was upgraded

Reply:
Ah okay, thanks